src/components/reusable/table/table.stories.ts

```diff
@@ -223,7 +223,9 @@ export const ExpandableRows: Story = {
               <kyn-expanded-tr .colSpan=${8}>
                 <div class="center-content">
                   <!-- Put your expanded table content here -->
-                  <h4>Expanded content goes here</h4>
+                  <kd-button>
+                    <h4>Expanded content goes here</h4>
+                  </kd-button>
```

Comment:
Any particular reason this example was turned into a button?

Reply:
Reverted